### PR TITLE
STU-1225-additional-changes improve internal-link wiki-modal interactions

### PR DESCRIPTION
**Pull request by lucaatstudylydotcom**
closing wiki-modal opened from ckeditor awesomeplete preview will result in less wierd side-effects (handled more by webapp STU-1225-additional-changes branch)

### CODE REVIEW

File: src/ui/internalLinkUi.js [hunk 1]
```diff
@@ -214,7 +213,7 @@ export default class InternalLinkUi extends Plugin {
                 // If still in a link element, simply update the position of the balloon.
                 // If there was no link (e.g. inserting one), the balloon must be moved
                 // to the new position in the editing view (a new native DOM range).
-                this.balloon.updatePosition(this.getBalloonPositionData());
+                this.balloon.updatePosition(this.getBalloonPositionData()); //TODO improve this method, make it work like the one in ckeditor5-math
```
Comment (MartinSchmidtStudyly) [inline src/ui/internalLinkUi.js:216]: Is this still needed?

File: src/ui/internalLinkUi.js [hunk 2]
```diff
@@ -503,6 +501,8 @@ export default class InternalLinkUi extends Plugin {
             }
         });
 
+        //TODO: call stopListening on destroy
```
Comment (MartinSchmidtStudyly) [inline src/ui/internalLinkUi.js:504]: Did you implement the stop listening?

File: src/ui/internalLinkUi.js (hunk 1 above)
Comment (lucaatstudylydotcom) [reply src/ui/internalLinkUi.js:216]: yes, i will improve it in another moment

File: src/ui/internalLinkUi.js (hunk 2 above)
Comment (lucaatstudylydotcom) [reply src/ui/internalLinkUi.js:504]: no, but look at this:
https://stackoverflow.com/questions/14041042/backbone-0-9-9-difference-between-listento-and-on
i dont think calling stopListening is crucial

**Review by MartinSchmidtStudyly — APPROVED** [review]

**Pull request merged**